socorro/external/es/super_search_fields.py

```diff
@@ -380,9 +380,6 @@ def add_field_to_properties(properties, namespaces, field):
                 '_all': {
                     'enabled': False,
                 },
-                '_source': {
-                    'compress': True,
-                },
```

Comment:
Can you talk a bit about this? Looks like we want to store the source data compressed--probably to save space. Do the newer versions default to that? Does this affect our space estimations?